e2e/schemas/createSchema.test.ts

```diff
@@ -128,4 +133,77 @@ describe('#createSchema', function () {
     assertExtrinsicSuccess(eventMap);
     assert.notEqual(createSchemaEvent, undefined);
   });
+
+  it('should successfully create a schema v3 with name', async function () {
```

Comment:
what happens if you try to create a schema named ".bob"?

Reply:
It would fail. Since the number of test cases were high I've tested all those in unit tests. End to end tests usually are not supposed to cover all the test-cases specially if it's covered in the other tests.